site/gatsby-site/cypress/e2e/integration/incidents.cy.js

```diff
@@ -38,6 +40,8 @@ describe('Incidents', () => {
 
     cy.contains('label', 'Editors').next().find('[type="text"]').type('Test Editor{enter}');
 
+    cy.wait(10000);
```

Comment:
This wait should be instead a `timeout` in the `cy.get` that queries the toast below.